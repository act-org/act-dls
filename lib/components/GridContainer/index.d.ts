/**
 * @prettier
 */
import * as React from 'react';
import { Props as GridBaseProps } from '~/components/GridBase';
export declare type Props = GridBaseProps;
declare const GridContainer: React.FC<Props>;
export default GridContainer;
//# sourceMappingURL=index.d.ts.map