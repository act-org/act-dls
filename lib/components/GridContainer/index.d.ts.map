{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["../../../src/components/GridContainer/index.tsx"],"names":[],"mappings":"AAAA;;GAEG;AAEH,OAAO,KAAK,KAAK,MAAM,OAAO,CAAC;AAE/B,OAAiB,EAAE,KAAK,IAAI,aAAa,EAAE,MAAM,uBAAuB,CAAC;AAEzE,oBAAY,KAAK,GAAG,aAAa,CAAC;AAElC,QAAA,MAAM,aAAa,EAAE,KAAK,CAAC,EAAE,CAAC,KAAK,CAE4B,CAAC;AAEhE,eAAe,aAAa,CAAC"}