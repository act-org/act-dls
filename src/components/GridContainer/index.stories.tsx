/**
 * @prettier
 */

import * as React from 'react';
import { select } from '@storybook/addon-knobs';

import GridContainer from '.';

export const container = (): React.ReactElement<any> => (
  <GridContainer
    alignContent={select(
      'AlignContent',
      {
        Center: 'center',
        FlexEnd: 'flex-end',
        FlexStart: 'flex-start',
        SpaceAround: 'space-around',
        SpaceBetween: 'space-between',
        Stretch: 'stretch',
      },
      'center',
    )}
    /*
    justify={select(
      'Justify',
      {
        Center: 'center',
        FlexEnd: 'flex-end',
        FlexStart: 'flex-start',
        SpaceAround: 'space-around',
        SpaceBetween: 'space-between',
        SpaceEvenly: 'space-evenly',
      },
      'center',
    )}
    */
    style={{
      height: 100,
      width: 100,
    }}
  >
    <div>Grid Container</div>
  </GridContainer>
);

export default {
  component: GridContainer,
  parameters: {
    info: {
      text: 'This is the container variant of the Grid component.',
    },
  },
  title: 'Atoms/Grid',
};
