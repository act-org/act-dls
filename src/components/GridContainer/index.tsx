/**
 * @prettier
 */

import * as React from 'react';

import GridBase, { Props as GridBaseProps } from '~/components/GridBase';

export type Props = GridBaseProps;

const GridContainer: React.FC<Props> = (
  props: Props,
): React.ReactElement<any> => <GridBase container {...props} />;

export default GridContainer;
