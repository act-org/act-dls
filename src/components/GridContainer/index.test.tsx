/**
 * @prettier
 */

import * as React from 'react';

import render from '~/helpers/test/render';

import GridContainer from '.';

describe('GridContainer', () => {
  it('matches the snapshot', () => {
    const { container } = render(<GridContainer>GridContainer</GridContainer>);

    expect(container).toMatchSnapshot();
  });
});
