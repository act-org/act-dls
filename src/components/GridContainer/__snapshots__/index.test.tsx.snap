// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridContainer matches the snapshot 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container"
  >
    GridContainer
  </div>
</div>
`;
